lang-python/pycryptodome/autobuild/defines

```diff
@@ -6,3 +6,5 @@ PKGDES="Collection of cryptographic algorithms and protocols"
 
 PKGBREAK="pycrypto<=2.6.1-4"
 PKGREP="pycrypto<=2.6.1-4"
+
+ABTYPE=python
```

Comment:
Maybe try pep517? This is what we are trying to move towards.

Reply:
Fixed.